docs/index.md

```diff
@@ -38,13 +38,27 @@ state, and with no need for callbacks. It's a simple and powerful app model
 that lets you build rich UIs incredibly quickly. To learn more about how
 Streamlit works, see [Main concepts](main_concepts.md).
 
+#### Install and play with Streamlit
```

Comment:
This title is a bit confusing, because there's already a section called "Install Streamlit" below.